src/strategies/catalog-get-tokenuri/transformer.mjs

```diff
@@ -50,9 +50,16 @@ export function onLine(line) {
       owner: "0x489e043540ff11ec22226ca0a6f6f8e3040c7b5a",
       version,
       createdAt: metadata?.block?.number,
-      tokenId: metadata?.tokenId,
+      tokens: [
+        {
+          minting: {
+            transactionHash: metadata?.transactionHash,
```

Comment:
At least in this use case, it'd be nice if we could serialize a worker message, resolve the value and then have it added automatically here. https://github.com/neume-network/schema/issues/53

Reply:
Maybe it doesn't even need to be serialized but just added here